README.md

```diff
@@ -1 +1,261 @@
+scopeinstance.operators.io.operator-framework/scopeinstance-sample configured
+```
+
+Verify RoleBindings in all namespaces.
+
+```
+NAMESPACE   NAME         ROLE               AGE
+test        test-64hk7   ClusterRole/test   2m45s
+```
+
+In the end, remove all namespaces and check if it can create ClusterRoleBinding and remove rolebindings.
+
+```
+apiVersion: operators.io.operator-framework/v1
+kind: ScopeInstance
+metadata:
+  name: scopeinstance-sample
+spec:
+  scopeTemplateName: scopetemplate-sample
+```
+
+```
+$ kubectl apply -f config/samples/operators_v1_scopeinstance.yaml
+scopeinstance.operators.io.operator-framework/scopeinstance-sample configured
+```
+
+Verify ClusterRoleBinding and RoleBindings.
+
+```
+$ kubectl get rolebindings --all-namespaces 
+No resources found
+```
+
+ClusterRoleBinding is created as shown below
+
+```
+$ kubectl get clusterrolebindings
+NAME         ROLE               AGE
+test-mskl2   ClusterRole/test   50s
+```
+
+## Dependency and platform support
+
+### Go version
+
+Release binaries will be built with the Go compiler version specified in the [developer guide][dev-guide-prereqs].
+A Go Operator project's Go version can be found in its `go.mod` file.
+
+[dev-guide-prereqs]:https://sdk.operatorframework.io/docs/contribution-guidelines/developer-guide#prerequisites
+
+### Kubernetes versions
+
+Supported Kubernetes versions for your Operator project or relevant binary can be determined
+by following this [compatibility guide][k8s-compat].
+
+[k8s-compat]:https://sdk.operatorframework.io/docs/overview#kubernetes-version-compatibility
+
+### Platforms
+
+The set of supported platforms for all binaries and images can be found in [these tables][platforms].
+
+[platforms]:https://sdk.operatorframework.io/docs/overview#platform-support
+
+## Community and how to get involved
+
+- [Operator framework community][operator-framework-community]
+- [Communication channels][operator-framework-communication]
+- [Project meetings][operator-framework-meetings]
+
+## How to contribute
+
+Check out the [contributor documentation][contribution-docs].
+
+## License
+
+Operator SDK is under Apache 2.0 license. See the [LICENSE][license_file] file for details.
+
+[controller-runtime]: https://github.com/kubernetes-sigs/controller-runtime
+[license_file]:./LICENSE
+[of-home]: https://github.com/operator-framework
+[of-blog]: https://www.openshift.com/blog/introducing-the-operator-framework
+[operator-link]: https://kubernetes.io/docs/concepts/extend-kubernetes/operator/
+[sdk-docs]: https://sdk.operatorframework.io
+[operator-framework-community]: https://github.com/operator-framework/community
+[operator-framework-communication]: https://github.com/operator-framework/community#get-involved
+[operator-framework-meetings]: https://github.com/operator-framework/community#meetings
+[contribution-docs]: https://sdk.operatorframework.io/docs/contribution-guidelines/
+
```

Comment:
I'm a little wary of adding this section unless we are going to also add the corresponding information specifically for oria-operator. I don't want to reference to much SDK specific stuff here as it could be confusing for potential users and contributors.

Reply:
Updated according to suggestions.